runtime/queries/scheme/highlights.scm

```diff
@@ -1,182 +1,76 @@
 ;; A highlight query can override the highlights queries before it.
 ;; So the order is important.
 ;; We should highlight general rules, then highlight special forms.
-
-(number) @number
-(character) @character
-(boolean) @boolean
-(string) @string
-[(comment)
- (block_comment)] @comment @spell
-
-;; highlight for datum comment
+(number) @number(character) @character(boolean) @boolean(string) @string[
```

Comment:
Here you're eating newlines, breaking the query (not sure why, as it doesn't affect other queries)?

Reply:
This is also because it's being run with an outdated version of the query.